### DISCUSSION TITLE
Unsure if my request is approved

### DISCUSSION THREAD

**Discussion by JacquesWhite** [community/community#117000] [GitHub Education] [answered]
### Hi everyone,

I have applied for the educational pack and it was rejected probably because of bad quality images of my student id, but shortly after the first submission, on the site appeared one more request which is apparently approved. However I still neither have access to any benefits, nor an ability to reapply again.
![image](https://github.com/community/community/assets/37044054/decb5b29-e0e2-48f1-87c3-cffc9a0a3a7b)
Is there any possibility to connect with support? What should I do?

**Comment by samus-aran** [comment, marked as answer]
Hi @JacquesWhite  👋

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You were last verified as a "null" on...](https://github.com/orgs/community/discussions/111352#null-last-verified-on-date)

I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
